tests/foreman/api/test_subnet.py

```diff
@@ -63,8 +105,18 @@ def test_positive_add_parameter_with_multiple_values(self):
         :expectedresults: The parameter with multiple values should be saved
             in subnet
         """
+        subnet = entities.Subnet().create()
+        name = gen_string('alpha')
+        values = ', '.join(generate_strings_list(exclude_types=['html']))
```

Comment:
isn't it a single string in result? Shouldn't we use list of multiple values instead?

Reply:
okay, we cannot create two parameters with same name and different values, so I cannot understand idea of that test case. Does Satellite treat '33434, 546rty' as two different values?

Reply:
And why do we skip html type?

Reply:
No, I am giving that as one value but seaparated with comma(,). Consider an example of address of a place having commas.

Reply:
Skipping html because I dont think any customer will try such kind of value and that will be used as parameter anywhere in host.
Now most of the tests are ignoring html